applications/models.py

```diff
@@ -246,6 +246,18 @@ def reject_submission(self):
     def revert_decision(self):
         """Revert previous submission decision"""
 
+    @transition(
+        field=state,
+        source=[
+            AppStates.COMPLETE.value,
+            AppStates.AWAITING_PAYMENT.value,
+            AppStates.REFUNDED.value,
+        ],
+        target=AppStates.REFUNDED.value,
```

Comment:
Do we need to change anything like `complete` to include `REFUNDED` or others as sources?

Reply:
I guess that depends on whether we want to allow the user to pay for it again and re-enroll using the same application - @pdpinch?

Reply:
@pdpinch any thoughts on whether a user should be able to use the same application to make another payment and re-enroll after receiving a refund?

Reply:
If it happens that a user requests a refund and then chooses to make more payments, it's going to be such an edge case that I don't think we should plan for it.